docroot/modules/custom/civictheme_migrate/assets/migrations/content/migrate_plus.migration.civictheme_page_annotate.yml

```diff
@@ -0,0 +1,202 @@
+      type: string
+      max_length: 2048
+process:
+  nid:
+    -
+      plugin: skip_on_empty
+      source: alias
+      method: row
+      message: 'URL is empty'
+    -
+      plugin: migration_lookup
+      migration: civictheme_page
+      source: alias
+      no_stub: true
+    -
+      plugin: skip_on_empty
+      method: row
+  title:
+    - plugin: skip_on_empty
+      source: title
+      method: row
+      message: 'Title is empty'
+    - plugin: substr
+      length: 255
+  field_c_n_summary: summary
+  field_c_n_topics:
+    - plugin: explode
+      delimiter: ','
+      source: topics
+    - plugin: entity_generate
+      value_key: name
+      ignore_case: true
+      operator: STARTS_WITH
+      entity_type: taxonomy_term
+      bundle_key: vid
+      bundle: 'civictheme_topics'
+  field_c_n_vertical_space:
+    - plugin: get
+      source: vertical_spacing
+    - plugin: default_value
+      default_value: ''
+  field_c_n_hide_sidebar:
+    - plugin: get
+      source: hide_sidebar
+    - plugin: default_value
+      default_value: 0
+  field_c_n_thumbnail:
+    plugin: migration_lookup
+    migration: civictheme_media_image
+    source: thumbnail_id
+  field_c_n_show_last_updated:
+    - plugin: get
+      source: show_last_updated_date
+    - plugin: default_value
+      default_value: 0
+  field_c_n_banner_theme:
+    - plugin: get
+      source: banner_theme
+    - plugin: default_value
+      default_value: 'dark'
+  field_c_n_banner_title: banner_title
+  field_c_n_banner_type:
+    - plugin: get
+      source: banner_type
+    - plugin: default_value
+      default_value: 'default'
+  field_c_n_blend_mode:
+    - plugin: get
+      source: banner_blend_mode
+    - plugin: default_value
+      default_value: 'color'
+  field_c_n_custom_last_updated:
+    - plugin: format_date
+      source: last_updated_date
+      from_format: 'j M Y'
+      to_format: 'Y-m-d'
+      settings:
+        validate_format: false
+    - plugin: skip_on_empty
+      method: process
+  field_c_n_show_toc:
+    - plugin: get
+      source: show_toc
+    - plugin: default_value
+      default_value: 0
+  field_c_n_banner_hide_breadcrumb:
+    - plugin: get
+      source: hide_breadcrumb
+    - plugin: default_value
+      default_value: 0
+  field_c_n_banner_featured_image:
+    plugin: migration_lookup
+    migration: civictheme_media_image
+    source: featured_image_id
+  field_c_n_banner_background:
+    plugin: migration_lookup
+    migration: civictheme_media_image
+    source: banner_background_id
+  'path/pathauto':
+    plugin: default_value
+    default_value: 0
+  'path/alias': alias
+  field_c_n_components:
+    -
+      plugin: ct_content_component_create
+      source: 'content'
+      migration_lookup:
+        __default__:
+          no_stub: true
+          migration:
+            - civictheme_page
+
+destination:
+  plugin: 'entity:node'
+  default_bundle: civictheme_page
+  overwrite_properties:
+    - field_c_n_components
```

Comment:
All fields need not be added again if not being overwritten, Annotate works as a node update in this case.

Reply:
https://github.com/salsadigitalauorg/civictheme_source/pull/1037#discussion_r1142848308
Related to this comment. 
annotate means we run this migration after the initial one. 
base fields can be removed and keep only fields that need to be updated in the second run 
ie: we have already set values for them in base migration, and annotate is used only to update few fields.

Reply:
thank you @joshua-salsadigital 
this is exactly what I though.

@govindmaloo 
could you please remove the base fields